packages/clerk-js/src/core/services/authentication/SessionCookieService.ts

```diff
@@ -64,7 +64,11 @@ export class SessionCookieService {
     try {
       this.updateSessionCookie(await this.getNewToken());
     } catch (e) {
-      return this.handleGetTokenError(e);
+      try {
+        return this.handleGetTokenError(e);
+      } catch {
+        return this.clerk.setActive({ session: null });
```

Comment:
If I'm not mistaken, there's already logic that handles the unauthenticated state here: 
https://github.com/clerkinc/javascript/blob/3b21b60091afe52a3276621c48142facb04c62d0/packages/clerk-js/src/core/resources/Base.ts#L66

Do we need to add more error codes to the above line? 
Can we avoid the need of this nested try/catch clause?

Reply:
Yeah, works well. Can't get rid of the try catch block though.